### DISCUSSION TITLE
Downloaded music location

### DISCUSSION THREAD

**Discussion by FloppexGaming** [jagrosh/MusicBot#1473] [Questions] [answered]
So I'd like to know where are the files located, because I don't want to fill my hard disk fully and someday move the bot on raspberry pi. I tried to look in My Documents, where the bot's folder is located, but there are only bot jar file and config.txt

**Comment by MichailiK** [comment, marked as answer]
Music does not get saved to storage. As the song is playing, JMusicBot slowly streams the music from YouTube into RAM & discards played sections afterwards. It's just like how playing on YouTube on your browser/phone works.

**Reply by FloppexGaming** [reply]
good to know, thank you
